### PR TITLE
Remove Reverse op from nGraph Python API

### PR DESCRIPTION
**Pull request by Ewa21**
The Reverse operation is not supported by the plugins.

### CODE REVIEW

File: ngraph/python/src/ngraph/opset3/__init__.py [hunk 1]
```diff
@@ -114,7 +114,6 @@
 from ngraph.opset1.ops import relu
 from ngraph.opset1.ops import reshape
 from ngraph.opset1.ops import result
-from ngraph.opset3.ops import reverse
```
Comment (GlebKazantaev) [inline ngraph/python/src/ngraph/opset3/__init__.py:117]: Does this lists with operations in inopsetX/__init__.py files match C++ opsetX_tbl.hpp ?

Comment (Ewa21) [reply ngraph/python/src/ngraph/opset3/__init__.py:117]: Opsets in pyAPI represent lists of operations from the specification in `docs/ops/opsetX.md`.

Comment (GlebKazantaev) [reply ngraph/python/src/ngraph/opset3/__init__.py:117]: @lazarevevgeny I remember you said that list of ops in ngraph opset is a reference list. So what should pyAPI follow? spec or list of ops in _tbl.hpp  files?

Comment (lazarevevgeny) [reply ngraph/python/src/ngraph/opset3/__init__.py:117]: The pyAPI should follow the opsetX_tbl.hpp file, not the documentation

**Pull request merged**